Prometheus.md

```diff
@@ -0,0 +1,75 @@
+﻿## Monitoring - Logging - Tracing 
+
+### Logging :
+
+Được sử dụng với mục đích ghi lại các  bản ghi chứa các mốc thời gian về những sự kiện và dữ liệu liên quan đến nó .
+Logging có 3 dạng bản ghi chính :
+- Plaintext : bản ghi là dạng văn bản tự do  ( dạng bản ghi được sử dụng phổ biến nhất )
+- Structured : thông thường được định dạng JSON 
+- Binary 
+Logging cung cấp cái nhìn sâu sắc có giá trị cùng với bối cảnh rộng lớn  đặc biệt hữu ích cho việc phát hiện các hành vi mới nổi và không thể đoán trước . Nhưng vấn đề có được, chuyển, lưu trữ và phân tích logs khá tốn kém vì đó nó chỉ log những gì cần thiết, có thể lưu được.
+
+### Tracing :
+
+Đại diện cho các bản ghi, cấu trúc dữ liệu của trace gần giống như sự kiện log. Một trace có thể cung cấp khả năng hiển thị vào cả đường dẫn (hành trình ) đi qua của một yêu cầu cũng như cấu trúc của yêu cầu.
+
+### Monitoring :
+
+Việc giám sát thu thập, tổng hợp và phân tích các số liệu đưa ra các cảnh báo cho các nhà phát triển khi một hệ thống không hoạt động như bình thường.  
+Matric là một biểu diễn số của dữ liệu được đo theo các khoảng thời gian. Các số liệu có thể khai thác sức mạnh của mô hình toán học và dự đoán để rút ra hiểu biết về hành vi của một hệ thống trong các khoảng thời gian trong hiện tại và tương lai.
```

Comment:
Metric

Reply:
Việc giám sát thu thập, tổng hợp và phân tích các số liệu đưa ra các cảnh báo cho các nhà phát triển khi một hệ thống không hoạt động như bình thường.  
Metric là một biểu diễn số của dữ liệu được đo theo các khoảng thời gian. Các số liệu có thể khai thác sức mạnh của mô hình toán học và dự đoán để rút ra hiểu biết về hành vi của một hệ thống trong các khoảng thời gian trong hiện tại và tương lai.